core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -158,7 +158,7 @@ inline void check_shmem_request(CudaInternal const* cuda_instance, int shmem) {
   }
 }
 
-template <class KernelFuncPtr>
+template <class DriverType, class LaunchBounds, class KernelFuncPtr>
```

Comment:
I don't understand what the two template parameters you added are here for.  They are not being used.

Reply:
they are there so the static variable is unique for every combo of DriverType and LaunchBounds. KernelFuncPtr is often just void (*)()

Reply:
I see.  Please comment